lib/dynflow/persistence_adapters/abstract.rb

```diff
@@ -36,6 +36,10 @@ def find_execution_plans(options = {})
         raise NotImplementedError
       end
 
+      def delete_execution_plans(filters, batch_size = 1000)
+        raise NotImplementedError
+      end
```

Comment:
What is the `filters` format? Could you add small example here or in the documentation?